api/all/src/main/java/io/opentelemetry/api/trace/ImmutableSpanContext.java

```diff
@@ -14,21 +14,30 @@
 abstract class ImmutableSpanContext implements SpanContext {
 
   static final SpanContext INVALID =
-      create(
+      createInternal(
           TraceId.getInvalid(),
           SpanId.getInvalid(),
           TraceFlags.getDefault(),
           TraceState.getDefault(),
           /* remote= */ false);
 
+  private static AutoValue_ImmutableSpanContext createInternal(
+      String traceId, String spanId, TraceFlags traceFlags, TraceState traceState, boolean remote) {
+    return new AutoValue_ImmutableSpanContext(
+        traceId, spanId, traceFlags, traceState, /* remote$= */ remote);
```

Comment:
Consider to remove the comment `/* remote$= */` not needed

Reply:
it always baffles me when errorprone will complain about this, and when it won't. I'm sure I added it because errorprone complained, but there might have been some refactoring after that.